os/include/tinyara/arch.h

```diff
@@ -2179,6 +2179,15 @@ void up_wdog_keepalive(void);
 
 #endif
 
+/****************************************************************************
+ * Name: is_kernel_text_space
+ *
+ * Description:
+ *   Check the address is in kernel text space or not
```

Comment:
Please add if after check. "Check if the address is in kernel text space or not"